docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '2'
+
+services:
+  web:
+    build: gob/web
+    container_name: web
+    ports: ["8080:8080"]
+    command: -gen-addr=linkerd:4140 -word-addr=linkerd:4140
+
+  gen:
+    build: gob/gen
+    container_name: gen
+    ports: ["8181:8181"]
+
+  word:
+    build: gob/word
+    container_name: word
+    ports: ["8282:8282"]
+
+  linkerd:
+    image: buoyantio/linkerd:0.5.0-SNAPSHOT
```

Comment:
We should probably point this at a released version.  Maybe hold off merging this until 0.6.0 is published (since the above configs use the 0.6.0 syntax).